README.md

```diff
@@ -74,6 +74,122 @@ Response headers:
     X-RateLimit-Reset: 1394506274
     X-RateLimit-Bucket: authorized-users
 
+## [Configuring Sphinx](./example.yaml)
+
+Rate limiting in Sphinx is managed by setting up `limits` in `yaml` configuration file. 
+Details about the configuration format can be found in the `[annotated example](./example.yaml)`
+
+It is important to understand the concept of `buckets` and `limits` to effectively configure a rate limiter.
+
+_Limit_: Limits define rate limiting policies that Sphinx enforces by counting requests in named _buckets_
+_Bucket_ A bucket is simply a named value. Each request that matches a limit increments the value of one bucket.
+
+Below is an example of a limit and three requests that increment two bucket values.
+
+### Test Limit
```

Comment:
It looks like we do below, but I might like to put it here. Something like "If you want this limit, here's how you would do it, and here's how these three requests would behave"